maturity-level-two/tests/Codit.UnitTest/Controllers/carControllerTest.cs

```diff
@@ -0,0 +1,50 @@
+using Codit.LevelTwo;
+using Codit.LevelTwo.Controllers.v1;
+using Codit.LevelTwo.Models;
+using Codit.LevelTwo.Services;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+using Codit.UnitTest;
+
+namespace Codit.UnitTest.Controllers
+{
+    public class CarControllerTest
+    {
+        private readonly CarController _controller;
+
+        public CarControllerTest()
+        {
+            _controller = new CarController(new CoditoRepositoryFake());
+            AutoMapperConfig.Initialize();
```

Comment:
Automapper should be initialized only once during the lifetime of the application, so it should not be called in the ctor of a test.  
You could create an xunit fixture that does this.